src/components/Article.tsx

```diff
@@ -0,0 +1,25 @@
+import React from "react";
+import { useNavigate } from "react-router-dom";
+
+interface ArticleProps {
```

Comment:
it like not article props, but its general model for article, so pleas extract this interface in models folded